### ISSUE TITLE
Geomap: Text box/label/annotation instead of point/symbol

### ISSUE THREAD

**Issue by slosst** [wet-boew/wet-boew#9805]
## Describe the purpose of this issue
I am looking for some guidance to implement a text box rendered in an overlay.

I have tried a variety of methods using documentation from OpenLayers but haven't had any results.

For example, I thought something like this in my overlay code would work (for each unique Location display the associated Value):

"style": {
                "type": "unique",
                "field": "location",
                "init": {
                    "label": "value"
                }
            }

**Comment by mercury64** [comment]
This is exactly what the first Dynamic Map Example Inline Data Layer does.
https://wet-boew.github.io/v4.0-ci/demos/geomap/geomap-en.html#dynamic

It has points, symbols, mouse over tool tip to show population, and a text box popup with all information.

From a UX point of view, landing on a page and having every popup open all at once would be a bad design.

This is a copy/paste from the examples:
```javascript
style: {
                type: "unique",
                field: "Type",
                init: {
                    Coal: {
                        externalGraphic:
                            "data:image/png;base64,iVBORw0KGgoA...I=",
                        fillOpacity: "1",
                        graphicWidth: "25",
                        name: "Coal mines",
                    },
                    "Metal mines and mills": {
                        externalGraphic:
                            "data:image/png;base64,iVBORw0...C",
                        fillOpacity: "1",
                        graphicWidth: "25",
                        name: "Metal mines",
                    }
}
```

**Comment by slosst** [comment]
Yes I am familiar with [Dynamic Map Example Inline Data Layer](https://wet-boew.github.io/v4.0-ci/demos/geomap/geomap-en.html#dynamic) and have had success implementing the structure from that example, however I was interested to see if it was possible to render text in the place of the externalGraphic element.

Notably the example provided still requires user interaction to display the underlying data.

Consider a usecase that would display one data point on each province (compared to the KML example that has hundreds of data points scattered across the country).

I agree that applying using popups in this manner is not appropriate, hence why I am investigating other options.

**Comment by mercury64** [comment]
You want something like this, Rich Text Labels:
https://openlayers.org/en/latest/examples/rich-text-labels.html

**Comment by slosst** [comment]
Yes - exactly! I didn't come across this when I was reviewing the OpenLayers documentation.
